sample-videochat-webrtc/src/main/res/layout/fragment_audio_conversation.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id = "@+id/fragmentIncomeCall"
+    style="@style/MatchParent"
+    >
+
+    <LinearLayout
+        android:id="@+id/info_about_call"
+        style="@style/MatchWidth"
+        android:orientation="vertical"
+        android:gravity="center_horizontal"
+        android:layout_marginTop="@dimen/action_bar_height"
+        >
+
+        <ImageView
+            android:id="@+id/caller_avatar"
+            style="@style/CallerAvatar"
+            />
+
+        <TextView
+            android:id="@+id/caller_name"
+            style="@style/WrapContent"
+            android:textColor="@color/caller_name_text_color"
+            android:textSize="@dimen/caller_name_text_size"
+            android:text="User Name"
+            android:maxLines="1"
+            android:maxLength="25"
+            android:paddingTop="20dp"
+            />
+
+        <Chronometer
+            android:id="@+id/timer_chronometer_audio_call"
+            style="@style/WrapContent"
+            android:layout_gravity="center"
+            android:layout_margin="@dimen/margin_common"
+            android:textColor="@color/also_on_call_text_color"
+            android:textSize="@dimen/text_size"
+            android:visibility="invisible" />
+
+        <TextView
+            android:id="@+id/also_on_call"
```

Comment:
https://github.com/QuickBlox/CodeStyle-Guidelines-for-Developers/blob/master/android/Resources.md#id-naming